admin_panel/viewset.py

```diff
@@ -11,10 +11,19 @@
     CourseViewset,
 )
 
+from .subjects.viewset import (
+    SubjectViewset
+)
+
+from .batches.viewset import (
+    BatchViewset
+)
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider consolidating university module imports</b></div>
<div id="fix">

Consider consolidating the imports from university-related modules (`subjects`, `batches`) into a single import statement from a centralized university package to improve code organization and maintainability.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -13,8 +13,4 @@
 from .courses.viewset import (
     CourseViewset,
 )
 from .viewsets.university import SubjectViewset, BatchViewset
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/sleep-404/ExamEvoke/pull/4#issuecomment-2499321783>#538c16</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged